src/components/Info.js

```diff
@@ -0,0 +1,116 @@
+import { useLocation } from 'react-router-dom';
+import { FaChevronDown, FaChevronUp, FaBitcoin } from 'react-icons/fa';
+import './Info.css';
+
+const Information = () => {
```

Comment:
- [ ] Kindly improve the styling on the details page. The values of the information should be aligned to the right, while the keys are aligned to the left. Right now, everything is aligned to the left. Kindly fix that.

- [ ] **[OPTIONAL]** While you are free to choose your colors I strongly advise against this color combination you currently have. Items on the page appear dim and one has to stress their vision to view the page.

  Since you are using black for font color I suggest you use a lighter background color, instead of this dark green color.

  ![djdkdo](https://github.com/yegonkimutai/React-capstone/assets/81468675/7c8d62c3-1db2-424a-a98d-13170419ca2e)